src/test/unit/service/subscriptionService.test.ts

```diff
@@ -39,5 +108,144 @@ describe('generate rows functions with subscriptions', () => {
     const courtSubscription= subscriptionService.getSubscriptionUrnDetails(validUrn);
     expect(courtSubscription).not.toBeNull();
   });
+});
+
+describe('handleNewSubscription function', () => {
+  it('should add new case subscription', async () => {
+    const pendingSubscription = {'hearing-selections[]': 'T485914'};
+    await subscriptionService.handleNewSubscription(pendingSubscription, '1');
+  });
+
+  it('should add new case subscriptions', async () => {
+    const pendingSubscription = {'hearing-selections[]': ['T485914', 'T485912']};
```

Comment:
need some tests that have `case-number` and `hearing-selections[]`

Reply:
there are no cases where body data will be both, it will be only one of these 3:
- urn
- hearing-selections[]
- court-selections[]